integration-tests/bats/sql.bats

```diff
@@ -39,71 +39,576 @@ teardown() {
     teardown_common
 }
 
-@test "sql: dolt sql -q without privilege file doesn't persist" {
-    # mysql database exists and has privilege tables
-    run dolt sql -q "show tables from mysql;"
-    [ "$status" -eq 0 ]
-    [[ "$output" =~ "user" ]] || false
-    [[ "$output" =~ "role_edges" ]] || false
+@test "sql: dolt sql -q" {
```

Comment:
This needs a better test name, this is doing user checks, not a sanity check on `dolt sql -q`